config/jobs/image-pushing/k8s-staging-cloud-provider-gcp.yaml

```diff
@@ -58,3 +58,32 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-cloud-provider-gcp-gcb
               - --env-passthrough=PULL_BASE_REF
               - .
+  kubernetes/cloud-provider-gcp:
+    - name: post-cloud-provider-gcp-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: provider-gcp-postsubmits, sig-k8s-infra-gcb
+        testgrid-tab-name: post-image-k8s-staging-cloud-provider-gcp
+      decorate: true
+      branches:
+        # For publishing canary images.
```

Comment:
what _aren't_ we triggering on?
for example, this will not work if the repo goes master=>main, it might be better to just run on every merge / tag

Reply:
Good question, this was copy/pasted the above examples. But I think at some point I would want to remove the master/main trigger, for now it's there for testing purposes